### PR TITLE
Transitioning array job to work on slurm

### PR DESCRIPTION
**Pull request by kathrynline**

### CODE REVIEW

File: R/pdf_array_job.R [hunk 1]
```diff
@@ -42,15 +42,16 @@ pdf_array_job <- function(jobs_file, r_script, final_out_dir,
...
-    " -P ", project, 
-    " -N ", job_name, 
-    " -now no",
-    " -o ", output, 
-    " -e ", errors,
+    "sbatch", 
+    " --mem ", fmem,
+    " -c ", fthread, 
+    " -t ", h_rt, 
+    " --array=1-", nrow(jobs),
```
Comment (miker985) [inline R/pdf_array_job.R:49]: This allows for all array jobs to be launched simultaneously (assuming no other cluster constraints)

I'm not sure if this is a problem or not.

If you want to add a max limit you do so with this arg. "--array=0-15%4" will create an array of 16 jobs limited to 4 running simultaneously

File: R/pdf_array_job.R [hunk 2]
```diff
@@ -4,32 +4,32 @@
...
+#' @param output Path to write cluster output messages to. Defaults to "/ihme/temp/slurmoutput/USERNAME/output"
+#' @param errors Path to write cluster error messages to. Defaults to "/ihme/temp/slurmoutput/USERNAME/errors"
 #' @param remove_temp_files Remove temporary files after jobs have completed? Defaults to TRUE.
 #' @param args Additional arguments to pass to array job launch script. 
 pdf_array_job <- function(jobs_file, r_script, final_out_dir, 
                           write_file_name="all_jobs.pdf",
-                          rshell="/share/singularity-images/lbd/shells/singR.sh -e s", 
-                          fthread=5, fmem='10G', h_rt="00:02:00:00", queue="all.q",
+                          rshell="/ihme/singularity-images/rstudio/shells/execRscript.sh -s", 
+                          fthread=5, fmem='10G', h_rt="02:00:00", queue="all.q",
```
Comment (stevebachmeier) [inline R/pdf_array_job.R:21]: Is this worth sticking on slurm's shiny new d.q?

Comment (kathrynline) [reply R/pdf_array_job.R:21]: I'm not sure how long that's going to stick around, and I'm worried I'll forget about it :(

File: R/pdf_array_job.R (hunk 1 above)
Comment (kathrynline) [reply R/pdf_array_job.R:49]: It's a good question... it's never been a concern before. I think generally people use this to launch one job per location in the hierarchy, which is a manageable amount of jobs.

**Pull request merged**